javascript/i18n/phonenumbers/phonenumberutil.js

```diff
@@ -20,11 +20,10 @@
  * Functionality includes formatting, parsing and validation.
  * (based on the java implementation).
  *
- * NOTE: A lot of methods in this class require Region Code strings. These must
- * be provided using ISO 3166-1 two-letter country-code format. These should be
- * in upper-case (but for compatibility lower-case is also allowed). The list of
- * the codes can be found here:
- * http://www.iso.org/iso/english_country_names_and_code_elements
+ * NOTE: A lot of methods in this class require Region Code strings. These must be provided using
+ * CLDR two-letter region-code format. These should be in upper-case. The list of the codes
```

Comment:
You wouldn't, as a user, provide 001: you'd use the appropriate method to get info for non-geographical entities (maybe there is an exception but I can't think of it)